java/HealthVault Mobile/HVSample/src/main/res/layout/add_medication.xml

```diff
@@ -0,0 +1,209 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+	android:layout_width="match_parent"
+	android:layout_height="match_parent">
+
+	<!-- Name Labels/Text -->
+
+	<TextView
+		android:id="@+id/nameLabel"
+		android:layout_width="69dp"
+		android:layout_height="20dp"
+		android:layout_marginTop="20dp"
```

Comment:
We are using a lot of common dimensions in this layout for margins, width and height. We should define common resources in dimens.xml and reference here